Hotails/urls.py

```diff
@@ -14,10 +14,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from main import views
+from django.urls import path, include
 
 urlpatterns = [
+
     path('admin/', admin.site.urls),
-    path('', views.homepage, name='homepage')
+    path('', include('main.urls')),
```

Comment:
Django will fire the first view for which the URL matches. It thus evaluates the URL top-to-bottom.
So when you have same 2 URL patterns, the bottom one is useless.

I suggest you switch them over and include the "homepage" in main.urls.